index.fr.md

```diff
@@ -0,0 +1,83 @@
+
+github:
+  repository: w3c/wai-design-develop-overview
+  path: "index.fr.md"
+
+feedbackmail: wai@w3.org
+class: tight-page
+footer: > # Text in footer in HTML
+  <p><strong>Date :</strong> Mise à jour : 20 septembre 2019.</p>
+  <p><strong>Rédactrice :</strong> <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
+  <p>Développé par le <em>Groupe de travail Éducation et Promotion</em> (<a href="http://www.w3.org/WAI/EO/">EOWG</a>).</p>
+ref: /design-develop/
+---
+
+{::nomarkdown}
+{% include box.html type="start" title="Résumé" class="" %}
+{:/}
+
+Règles pour l’écriture, la conception, et le développement dans le contexte de l’accessibilité.
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+
+{% include toc.html %}
+
+## Conseils pour démarrer dans l’accessibilité Web  {#tips}
+
+Les conseils suivants introduisent les éléments essentiels à prendre en considération pour rendre votre site Web plus accessible aux personnes handicapées. Ils fournissent des liens vers des ressources supplémentaires.
+
+[L’écriture dans le cadre de l’accessibilité Web](/tips/writing/)
+: Conseils pour écrire et présenter du contenu.
+
+[la conception dans le cadre de l’accessibilité Web](/tips/designing/)
+: Conseils pour l’interface utilisateur et la conception visuelle.
+
+[Le développement dans le cadre de l’accessibilité Web](/tips/developing/)
+: Conseils pour le balisage et le code.
+{:.paragraph-like}
+
+## Ressource multimédia pour l’audio et la vidéo
+
+[Rendre le contenu multimédia audio et vidéo accessible](https://www.w3.org/WAI/media/av/)
+: Vous aide à comprendre et à créer des légendes et des sous-titres, des audiodescriptions des informations visuelles, des transcriptions descriptives et le langage des signes pour le contenu multimédia. Inclut des conseils pour créer de nouvelles vidéos et en savoir plus sur l’accessibilité des lecteurs multimédia. Introduit les expériences utilisateur et les avantages pour les organisations.  
+{:.paragraph-like}
+
+## Tutoriels sur l’accessibilité Web {#tutorials}
+
+Les tutoriels donnent des conseils pour la création de composants répondant aux exigences des Règles pour l’accessibilité des contenus Web (WCAG), dans le but de les rendre plus accessibles pour les personnes en situation de handicap et de fournir une meilleure expérience utilisateur pour tous.
+
+* **[Tutoriel : structure d’une page](https://www.w3.org/WAI/tutorials/page-structure/)**(en anglais)
+* **[Tutoriel : menus](https://www.w3.org/WAI/tutorials/menus/)**
+* **[Tutoriel : images](https://www.w3.org/WAI/tutorials/images/)**
+* **[Tutoriel : tableaux](https://www.w3.org/WAI/tutorials/tables/)**
+* **[Tutoriel : formes](https://www.w3.org/WAI/tutorials/forms/)**
+* **[Tutoriel : carrousels](https://www.w3.org/WAI/tutorials/carousels/)**
+
+## Autres ressources pour la conception et le développement
+
+[Comment répondre aux exigences des WCAG (référence rapide)](https://www.w3.org/WAI/WCAG21/quickref/)
```

Comment:
What do you think of "aide-mémoire" rather than "référence rapide" ? (see comment below)